rq/worker.py

```diff
@@ -225,6 +218,16 @@ def __init__(self, queues, name=None, default_result_ttl=DEFAULT_RESULT_TTL,
 
         self.disable_default_exception_handler = disable_default_exception_handler
 
+        if prepare_for_work:
+            self.hostname = socket.gethostname()
+            self.pid = os.getpid()
+            connection.client_setname(self.name)
+            self.address = [client['addr'] for client in connection.client_list() if client['name'] == self.name][0]
```

Comment:
Can we change `address` to `ip_address`?

Reply:
done